### DISCUSSION TITLE
Does Filament support Eloquent Collection?

### DISCUSSION THREAD

**Discussion by rezaulhreza** [filamentphp/filament#3574] [Help] [answered]
Filament supports Eloquent builder out of the box.
However, on certain occasions I tried to use Eloquent Collection and I was getting error saying that it is expecting Eloquent Builder.

**Comment by ryangjchandler** [comment, marked as answer]
Filament only supports queries yes. If you want to use a collection or array, Sushi might interest you.

**Reply by rezaulhreza** [reply]
Can it be expected in v3?

**Reply by danharrin** [reply]
It works fine in v2 with Sushi, theres no room for improvement in v3
